.packages

```diff
@@ -0,0 +1,91 @@
+# Generated by pub on 2019-08-07 08:43:13.263408.
```

Comment:
@olesiathoms-wk could you add this file to the `.gitignore` as well and then uncommit it via `git rm .packages`?